### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'value'

### ISSUE THREAD

**Issue by flexwang** [microsoft/DeepSpeed-MII#340]
Errors when generating for llama2-70B with tp=8, llama2-7B is fine though.
```
Traceback (most recent call last):
  File "/home/corvo/.local/lib/python3.10/site-packages/grpc/_server.py", line 552, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "/home/corvo/.local/lib/python3.10/site-packages/mii/grpc_related/modelresponse_server.py", line 96, in GeneratorReply
    return task_methods.pack_response_to_proto(responses)
  File "/home/corvo/.local/lib/python3.10/site-packages/mii/grpc_related/task_methods.py", line 85, in pack_response_to_proto
    finish_reason=str(r.finish_reason.value),
AttributeError: 'NoneType' object has no attribute 'value'
```

**Comment by mrwyattii** [comment]
Thank you for reporting this bug. Are you able to reliable reproduce this? If so, you can share your script so I can try to reproduce?

**Comment by mrwyattii** [comment]
This should be fixed in #350
